docs/config.json

```diff
@@ -171,6 +171,11 @@
               "title": "Migrating From Older Charts",
               "slug": "/deploy-a-cluster/helm-deployments/migration/",
               "forScopes": ["oss", "enterprise"]
```

Comment:
I wonder if we should remove this in v12. The `teleport` chart has been deprecated since v7 I think, it's highly unlikely that anyone is using it any more and if they are, we can link them to an older version of the docs.

Reply:
Roman mentioned we still had users running Teleport 6 and legacy charts. I suggest we should keep this section because we especially want those users to upgrade.

Reply:
😢